### PR TITLE
SAR-8846 Upgrade to Play 2.8

### PR DESCRIPTION
**Pull request by woollett-r**

### CODE REVIEW

File: project/AppDependencies.scala [hunk 1]
```diff
@@ -4,12 +4,12 @@ import sbt._
 object AppDependencies {
 
   val compile: Seq[ModuleID] = Seq(
-    "uk.gov.hmrc" %% "bootstrap-frontend-play-27" % "5.10.0",
-    "uk.gov.hmrc" %% "bootstrap-backend-play-27" % "5.10.0",
-    "uk.gov.hmrc" %% "play-frontend-govuk" % "0.71.0-play-27",
-    "uk.gov.hmrc" %% "simple-reactivemongo" % "8.0.0-play-27",
-    "uk.gov.hmrc" %% "play-frontend-hmrc" % "0.59.0-play-27"
-
+    "uk.gov.hmrc" %% "bootstrap-frontend-play-28" % "5.3.0",
```
[the remark this answers is not in the document]
Comment (woollett-r) [reply project/AppDependencies.scala:7]: Done

File: project/AppDependencies.scala [hunk 2]
```diff
@@ -4,12 +4,12 @@ import sbt._
...
+    "uk.gov.hmrc" %% "bootstrap-backend-play-28" % "5.3.0",
```
[the remark this answers is not in the document]
Comment (woollett-r) [reply project/AppDependencies.scala:8]: Done

File: project/AppDependencies.scala [hunk 3]
```diff
@@ -4,12 +4,12 @@ import sbt._
...
+    "uk.gov.hmrc" %% "play-frontend-govuk" % "0.71.0-play-28",
```
[the remark this answers is not in the document]
Comment (woollett-r) [reply project/AppDependencies.scala:9]: Done

File: project/AppDependencies.scala [hunk 4]
```diff
@@ -4,12 +4,12 @@ import sbt._
...
+    "uk.gov.hmrc" %% "simple-reactivemongo" % "8.0.0-play-28",
+    "uk.gov.hmrc" %% "play-frontend-hmrc" % "0.64.0-play-28",
```
[the remark this answers is not in the document]
Comment (woollett-r) [reply project/AppDependencies.scala:11]: Done

File: project/plugins.sbt [hunk 5]
```diff
@@ -6,7 +6,7 @@ addSbtPlugin("uk.gov.hmrc" % "sbt-auto-build" % "3.3.0")
 
 addSbtPlugin("uk.gov.hmrc" % "sbt-distributables" % "2.1.0")
 
-addSbtPlugin("com.typesafe.play" % "sbt-plugin" % "2.7.9")
+addSbtPlugin("com.typesafe.play" % "sbt-plugin" % "2.8.7")
```
[the remark this answers is not in the document]
Comment (woollett-r) [reply project/plugins.sbt:9]: Done

File: project/plugins.sbt [hunk 6]
```diff
@@ -6,7 +6,7 @@ addSbtPlugin("uk.gov.hmrc" % "sbt-auto-build" % "3.3.0")
```
[the remark this answers is not in the document]
Comment (woollett-r) [reply project/plugins.sbt:6]: Done

File: it/uk/gov/hmrc/incorporatedentityidentificationfrontend/assets/TestConstants.scala [hunk 7]
```diff
@@ -4,6 +4,8 @@ package uk.gov.hmrc.incorporatedentityidentificationfrontend.assets
 import play.api.libs.json.{JsObject, Json}
 import uk.gov.hmrc.incorporatedentityidentificationfrontend.models.BusinessEntity.{BusinessEntity, CharitableIncorporatedOrganisation, LimitedCompany, RegisteredSociety}
 import uk.gov.hmrc.incorporatedentityidentificationfrontend.models._
+import uk.gov.hmrc.incorporatedentityidentificationfrontend.models.BusinessVerificationStatus._
+import uk.gov.hmrc.incorporatedentityidentificationfrontend.models.RegistrationStatus._
 import uk.gov.hmrc.incorporatedentityidentificationfrontend.testonly.forms.TestCreateJourneyForm.businessVerificationCheck
```
[the remark this answers is not in the document]
Comment (woollett-r) [reply it/uk/gov/hmrc/incorporatedentityidentificationfrontend/assets/TestConstants.scala:9]: Done

**Review by refaque-ahmad — APPROVED** [review]
LGTM

**Pull request merged**